src/main/java/org/kohsuke/github/GHCreateRepositoryBuilder.java

```diff
@@ -211,4 +229,15 @@ public GHRepository create() throws IOException {
         return builder.method("POST").withUrlPath(apiUrlTail).fetch(GHRepository.class).wrap(root);
     }
 
+    /**
+     * Creates a repository with all the parameters, and with Preview BAPTISE for template repo.
+     *
+     * @return the gh repository
+     * @throws IOException
+     *             if repsitory cannot be created
+     */
+    public GHRepository createWithTemplate() throws IOException {
```

Comment:
Similar to my previous comment could we have a `fromTemplateRepository(String templateRepo, String templateOwner)` method?   I think the `create()` method could have `.withPreview(BAPTISE)` set in general without any negative effect, but I could be wrong.

Reply:
According to the  [official doc](https://developer.github.com/v3/repos/#create-a-repository-using-a-template) , I don't think we can add an `fromTemplateRepository(String templateOwner, String templateRepo)` to the builder, Because templateOwner and templateRepo are parts of the request url not parameters. anyway,  I will try it out, and find the result.
"create() method could have .withPreview(BAPTISE) set in general without any negative effect" —— I will try it later also, thanks for your idea.